apps/site/components/Downloads/ReleaseModal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import Modal from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import LinkWithArrow from '@/components/LinkWithArrow';
+import { BASE_CHANGELOG_URL } from '@/next.constants.mjs';
+import type { NodeRelease } from '@/types';
+import { getNodeApiLink } from '@/util/getNodeApiLink';
+
+import { MinorReleasesTable } from './MinorReleasesTable';
+
+type ReleaseModalProps = {
+  isOpen: boolean;
+  closeModal: () => void;
+  release: NodeRelease;
+};
+
+const ReleaseModal: FC<ReleaseModalProps> = ({
+  isOpen,
+  closeModal,
+  release,
+}) => {
+  const t = useTranslations('components.releaseModal');
+
+  return (
+    <Modal
+      heading={`Node.js ${release.major} Release`}
+      subheading={release?.codename ?? ''}
```

Comment:
perhaps reinstate the status to allow the user to always see this information. Especially if it's an EoL version.